File: android/src/main/java/dragon3/data/AnimeData.java
/*
 * 作成日: 2004/03/07
 */
package dragon3.data;

import dragon3.common.constant.AnimeType;

/**
 * @author k-saito
 */
@lombok.Data
public class AnimeData implements Data {

	private String id = "none";
	private String name = "none";
	private AnimeType type = AnimeType.SINGLE;
	private String image = "none.png";
	private int sleep;

	@Override
	public String toString(){
		return id + " - " + name;
	}
}

File: core/src/main/kotlin/dragon3/data/AnimeData.kt
/*
 * 作成日: 2004/03/07
 */
package dragon3.data

import dragon3.common.constant.AnimeType

/**
 * @author k-saito
 */
class AnimeData : Data {

    override var id = "none"
    override var name = "none"
    var type = AnimeType.SINGLE
    var image = "none.png"
    var sleep: Int = 0

    override fun toString(): String {
        return id + " - " + name
    }
}
